MoltenVK/MoltenVK/Commands/MVKCommandBuffer.h

```diff
@@ -372,6 +372,9 @@ class MVKCommandEncoder : public MVKBaseDeviceObject {
     /** Get a temporary MTLBuffer that will be returned to a pool after the command buffer is finished. */
     const MVKMTLBufferAllocation* getTempMTLBuffer(NSUInteger length, bool isPrivate = false, bool isDedicated = false);
 
+	/** Copy the bytes to a temporary MTLBuffer that will be returned to a pool after the command buffer is finished. */
+	const MVKMTLBufferAllocation* copyToTempMTLBufferAllocation(const void* bytes, NSUInteger length, bool isPrivate = false, bool isDedicated = false);
```

Comment:
I don't think either of these makes sense here. You wouldn't be able to access a `Private` buffer from the CPU to populate it in the first place, and `isDedicated` is only needed right now for the very particular use case of visibility result buffers. I guess adding `isDedicated` for future-proofing maybe might make sense, if it turns out to be needed for something, but I don't see `isPrivate` ever being useful here.

Reply:
Doh! 🤦🏻‍♂️ Brain fart on my part about `isPrivate`. Good catch. I've left the `isDedicated` for future options.